### PR TITLE
runbooks, Add info about KubeMacPoolDuplicateMacsFound runbook

### PR DESCRIPTION
**Pull request by oshoval**
Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

**Comment by oshoval** [comment]
/cc @RamLavi 

I will add the prints of vm / interface, mac that are mentioned, on the kubemacpool PR

**Comment by oshoval** [comment]
trying to generate the list

```
monitoring (dup) $ python automation/generate_pages_index.py
  File "automation/generate_pages_index.py", line 20
    lines = list(map(lambda f: f"* [{f[:-3]}]({f})\n", files))
                                                    ^
SyntaxError: invalid syntax
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:6]:
```suggestion
Upon starting KubeMacPool, it scans the cluster for MAC Addresses in VMs set on the managed namespaces.
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:7]:
```suggestion
In case it finds duplicates it will raise this alert.
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:5]:
```suggestion
KubeMacPool is responsible to allocate MAC Addresses and prevent MAC Address conflicts.
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:8]: This is not accurate, I would vote to remove this line.. we should keep the explanation generic and simple
```suggestion
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:12]:
```suggestion
Having a duplicate MAC Addresses can break the requirement that each interface on the same LAN,
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:13]:
```suggestion
should have a unique MAC Address.
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:17]: I would be more specific.. don't expect the sys-admin to know what's KMP's namespace
```
- Get kubemacpool-mac-controller pod namespace + command example
- Extract the conflicting VMs from kubemacpool pod logs, from which you will have 1, 2 and 3: + command example
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:29]:
```suggestion
Update the VMs to remove the duplicate MAC Addresses.
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:30]: please add command example

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:31]: I would remove this line, no further details needed imo.

**Review by RamLavi — CHANGES_REQUESTED** [review]

**Comment by oshoval** [comment]
Addressed comments, thanks

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:11]: can we rephrase this? something like 
```suggestion
Duplicate MAC Addresses on the same LAN may cause unexpected and various connectability issues in the network.
```

File: docs/runbooks/KubeMacPoolDuplicateMacsFound.md
Comment (RamLavi) [inline docs/runbooks/KubeMacPoolDuplicateMacsFound.md:17]: This line appears in the same line when looking in the `Display rich format` option, I think you need to add a new line. also in the rest

**Comment by oshoval** [comment]
Addressed comments

**Review by RamLavi — APPROVED** [review]
/lgtm

/hold
until we merge this alert on CNAO

**Comment by oshoval** [comment]
Thanks
@sradco can you take a look please?

**Comment by oshoval** [comment]
> @oshoval can we wait until the actual alert is added on [kubevirt/cluster-network-addons-operator#952](https://github.com/kubevirt/cluster-network-addons-operator/pull/952)?

shouldn't it be the opposite ?
but anyway we can merge that one already i think ?

**Comment by RamLavi** [comment]
/hold cancel
@sradco I'm good when you are ready to merge this PR

**Pull request merged**
